protocol/avdl/keybase1/saltpack.avdl

```diff
@@ -96,7 +96,7 @@ protocol saltpack {
 
   SaltpackEncryptFileResult saltpackEncryptStringToTextFile(int sessionID, string plaintext, SaltpackFrontendEncryptOptions opts);
 
-  SaltpackEncryptFileResult saltpackEncryptFile(int sessionID, string filename, SaltpackFrontendEncryptOptions opts);
+  SaltpackEncryptFileResult saltpackEncryptFile(int sessionID, string filename, string destinationDir, SaltpackFrontendEncryptOptions opts);
```

Comment:
// destinationDir (optional) If unset then the result will be put in the same directory as `filename`.

Reply:
i think you and I are the only ones that read avdl for the interface spec, but sure.

Reply:
Hey I do it too!